docs/pt/docs/tutorial/middleware.md

```diff
@@ -0,0 +1,61 @@
+# Middleware
+
+Você pode adicionar middleware à suas aplicações **FastAPI**.
+
+Um "middleware" é uma função que manipula cada **request** antes de ser processada por qualquer *operação de caminho* específica. E também cada **response** antes de retorná-la.
```

Comment:
"path operation" has been translated as "operação de rota" more often.

But it's not wrong...

I would translate request and response as "requisição" and "retorno" as well.

Something like:

*"Um "middleware" é uma função que manipula cada requisição antes de ser processada por qualquer *operação de rota* específica. E também cada retorno antes de devolvê-lo."*

Reply:
for sure. I'll fix it.

Reply:
I think is better translate response to "resposta" because we can translate sometthing like "Então ele retorna o **retorno**"